vizro-core/src/vizro/models/_utils.py

```diff
@@ -0,0 +1,122 @@
+import inspect
+from dataclasses import dataclass
+from typing import Any, Callable, Dict, List, Optional, Set, Tuple
+
+MODEL_NAME = "model_name"
+ACTIONS_CHAIN = "ActionsChain"
+ACTION = "actions"
+
+
+@dataclass
+class PathReplacement:
+    detect_path: str
+    replace_path: str
+    from_import: Callable
+
+
+@dataclass
+class CapturedCallableInfo:
+    name: str
+    module: str
+    args: List[Tuple[str, Any]]
```

Comment:
Maybe easier to keep this as a dict or whatever `CapturedCallable._arguments` returns?

Reply:
Sure can also do that